src/views/Dissolutions/ReviewConfirmDissolution.vue

```diff
@@ -508,4 +508,26 @@ export default class ReviewConfirmDissolution extends Mixins(DateMixin) {
     margin-left: -46px;
   }
 }
+
+::v-deep #AR-step-4-container, ::v-deep #document-delivery {
```

Comment:
Just curious about the first selector `#AR-step-4-container`, is this required here?

Reply:
Oh, is this a class brought in via a shared component maybe?

Reply:
Thats the id in Certify shared component

Reply:
dang :(

Reply:
Carry over from a translation from our Annual Report i suppose!

Reply:
Nice!

Reply:
Yeah, nice change. Thanks Vyaskh.